src/Microsoft.PowerShell.Commands.Utility/commands/utility/XmlCommands.cs

```diff
@@ -19,95 +19,31 @@
 namespace Microsoft.PowerShell.Commands
 {
     /// <summary>
-    /// implementation for the Export-Clixml command
+    /// Base implementation for ConvertTo-Clixml and Export-Clixml
```

Comment:
The comment is public - please add final dot.
Below please review formatting public comments too.